examples/all-clusters-app/ameba/README.md

```diff
@@ -137,28 +137,70 @@ to be On or Off.
 
 -   Open the USB-TTL serial port and type `help` to view the available commands
 
-## Binding and Controlling a Lighting Device
-
--   This example shows how to bind a Switch Device to a Lighting Device and
+-   To know what are the available subcommands and what it does, simply enter
+    the command that you are interested in
+
```

Comment:
use backticks here since these are shell things. This avoids the spellchecker trying to think these are words (like `offWE` and such are not english words but rather code.